internal/host/validator.go

```diff
@@ -927,7 +926,7 @@ func (v *validator) validateNetworkLatencyForRole(host *models.Host, clusterRole
 		for _, l3 := range r.L3Connectivity {
 			if l3.AverageRTTMs > *clusterRoleReqs.Total.NetworkLatencyThresholdMs {
 				if _, ok := failedHostIPs[l3.RemoteIPAddress]; !ok {
-					hostname, role, err := GetHostnameAndEffectiveRoleByIP(l3.RemoteIPAddress, hosts)
```

Comment:
Can we remove the old GetHostnameAndEffectiveRoleByIP func? Seems it would be unused after this change?

Reply:
It is still in use

Reply:
> It is still in use

Oh, I assumed only the validator is using it. Where is it in use elsewhere?